renderchan/utils.py

```diff
@@ -24,12 +24,9 @@ def is_exe(fpath):
         if is_exe(program):
             return program
     else:
-        for path in os.environ["PATH"].split(os.pathsep):
-            path = path.strip('"')
-            exe_file = os.path.join(path, program)
-            exe_file = os.path.realpath(exe_file)
-            if is_exe(exe_file):
-                return exe_file
+        path = shutil.which(program)
```

Comment:
shutil.which() should work fine on Windows. ^__^ - https://docs.python.org/3/library/shutil.html#shutil.which